Assignment1-JavaScript/palindrome.js

```diff
@@ -0,0 +1,70 @@
+
+function LinkedList(val){
+    this.val = val;
+    this.next = null;
+}
+
+function isPalindrome(x) {
```

Comment:
I'm leaving this at the top of the function, but please make your variable names actually descriptive of thier purpose. I see why they are named what they are, but this is bad conventionally.